site/docs/v1/tech/apis/consents.adoc

```diff
@@ -166,6 +167,61 @@ This API does not return a JSON response body.
 
 include::docs/v1/tech/apis/_standard-delete-response-codes.adoc[]
 
+== Search for Consents
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Consents and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
```

Comment:
I think you need a `=== Request` here

Reply:
I guess that's what happens when I copy-paste yours and that header was in between two comments 😅 

I'm guessing all my other PRs need this too.